dbschema.txt.js

```diff
@@ -123,71 +123,4 @@ requests_public = [{
     // Info object
     ownerInfo: { firstName, displayName, takenAt },
   }
```

Comment:
We may want to remove this file if we are going to be using the JSON schema moving forward. Guessing you left it in place while we are still transitioning?

Reply:
Correct, I haven't put in the time to move these yet.